.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ parameters:
     default: false
   dev-orb-version:
     type: string
-    default: "dev:alpha"
+    default: "dev:first"
```

Comment:
this will need to stay alpha. Pipeline parameters for this config are crucial for the development pipeline. The orb-tools orb relies on this for automatic publishing.